src/node/desktop/test/unit/main/browser-window.test.ts

```diff
@@ -17,25 +17,17 @@ import { describe } from 'mocha';
 import { assert } from 'chai';
 
 import { BrowserWindow } from 'electron';
-import { execSync, spawn } from 'child_process';
+import * as http from 'http';
```

Comment:
do we really need to import everything?

Reply:
Good point. It only needs `createServer`.